service/src/test/java/org/cbioportal/service/impl/MutationCountServiceImplTest.java

```diff
@@ -0,0 +1,187 @@
+
+        Mockito.when(mutationCountRepository.getPositionMutationCounts(org.mockito.Matchers.eq("BRAF"), org.mockito.Matchers.anyList())).thenReturn(brafResult);
+	Mockito.when(mutationCountRepository.getPositionMutationCounts(org.mockito.Matchers.eq("KRAS"), org.mockito.Matchers.anyList())).thenReturn(krasResult);
+	
+	List<PositionMutationCount> expectedResult = new ArrayList<>();
+	expectedResult.add(pmc1);
+	List<PositionMutationCount> result = mutationCountService.getPositionMutationCounts(testGenes, testPositions);
+	Assert.assertEquals(result, expectedResult);
+    }
+    
+     @Test
+    public void getPositionMutationCountsNoPositions() throws Exception {
+
+        ArrayList<String> testGenes = new ArrayList<>();
+	testGenes.add("BRAF");
+	testGenes.add("KRAS");
+        ArrayList<List<Integer>> testPositions = new ArrayList<>();
+
+	PositionMutationCount pmc1 = new PositionMutationCount();
+	PositionMutationCount pmc2 = new PositionMutationCount();
+        List<PositionMutationCount> brafResult = new ArrayList<>();
+	List<PositionMutationCount> krasResult = new ArrayList<>();
+	brafResult.add(pmc1);
+	krasResult.add(pmc2);
+
+        Mockito.when(mutationCountRepository.getPositionMutationCounts(org.mockito.Matchers.eq("BRAF"), org.mockito.Matchers.anyList())).thenReturn(brafResult);
+	Mockito.when(mutationCountRepository.getPositionMutationCounts(org.mockito.Matchers.eq("KRAS"), org.mockito.Matchers.anyList())).thenReturn(krasResult);
+	
+	List<PositionMutationCount> expectedResult = new ArrayList<>();
+	List<PositionMutationCount> result = mutationCountService.getPositionMutationCounts(testGenes, testPositions);
+	Assert.assertEquals(result, expectedResult);
+    }
+    @Test
+    public void getPositionMutationCountsNoGenes() throws Exception {
+
+        ArrayList<String> testGenes = new ArrayList<>();
+        ArrayList<List<Integer>> testPositions = new ArrayList<>();
+	ArrayList<Integer> brafPositions = new ArrayList<>();
+	ArrayList<Integer> krasPositions = new ArrayList<>();
+	brafPositions.add(1);
+	brafPositions.add(2);
+	krasPositions.add(3);
+	testPositions.add(brafPositions);
+	testPositions.add(krasPositions);
+
+	PositionMutationCount pmc1 = new PositionMutationCount();
+	PositionMutationCount pmc2 = new PositionMutationCount();
+        List<PositionMutationCount> brafResult = new ArrayList<>();
+	List<PositionMutationCount> krasResult = new ArrayList<>();
+	brafResult.add(pmc1);
+	krasResult.add(pmc2);
+
+        Mockito.when(mutationCountRepository.getPositionMutationCounts(org.mockito.Matchers.eq("BRAF"), org.mockito.Matchers.anyList())).thenReturn(brafResult);
+	Mockito.when(mutationCountRepository.getPositionMutationCounts(org.mockito.Matchers.eq("KRAS"), org.mockito.Matchers.anyList())).thenReturn(krasResult);
+	
+	List<PositionMutationCount> expectedResult = new ArrayList<>();
+	List<PositionMutationCount> result = mutationCountService.getPositionMutationCounts(testGenes, testPositions);
```

Comment:
Here, we passed in no genes, but two position lists .. and we expect nothing out.
This also confuses me.